src/cli.py

```diff
@@ -29,6 +30,18 @@ def tokens(count, skip, sort, sort_direction, format, verbose):
     get_tokens(count=count, skip=skip, sort=sort, sort_direction=sort_direction, format=format, verbose=verbose)
 
 
+@main.command()
+@click.option('--count', default=100, help='Number of prices to return, used for pagination')
+@click.option('--skip', default=0, help='Number of prices to skip, used for pagination')
+@click.option('--sort', default="batchId", help='Sort result by a field, used for pagination')
+@click.option('--sort-direction', default="asc", help='Sort direction. "asc" for ascending, "desc" for descending order')
+@click.option('--format', default="pretty", help='Format type i.e. pretty, csv')
+@click.option('-v', '--verbose', count=True)
```

Comment:
Do you have to duplicate this for every command?

Would it work if you have a `main` command and branch out to each specific sub command?

Reply:
I'm not sure honestly!
Let's do a issue for this :) 
For now, it's not bothering much, but ideally we could do some refactor, specially for the pagination and verbose flags.

https://github.com/gnosis/dex-cli/issues/40